.github/workflows/publish_collection.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish fedora.linux_system_roles collection to Ansible Galaxy
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 0 * * *'
+
+jobs:
+  publish_collection:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout auto-maintenance repository
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+
+      - name: Install dependencies
+        shell: bash
+        run: |
+          set -ex
+          pip install --upgrade pip
+          pip install --upgrade ansible ansible-galaxy galaxy-importer
```

Comment:
Could you please add `pypandoc` and `rst2html` to the `pip install` command line? Thanks!

Reply:
This should be done now.